mne/baseline.py

```diff
@@ -55,15 +59,12 @@ def rescale(data, times, baseline, mode='mean', copy=True, verbose=None):
     data_scaled: array
         Array of same shape as data after rescaling.
     """
+    _log_rescale(baseline, mode)
+    if baseline is None:
+        return data
```

Comment:
This changes the previous reasonable behavior of the function -- before if `copy is True` then it would always return a copy, now it does in some cases and doesn't in others.